src/fusion_pipeline.py

```diff
@@ -45,15 +45,21 @@ def process_cloud(self, tilecode, points, mask=None):
         An array of shape (n_points,) with dtype=uint16 indicating the label
         for each point.
         """
-        if mask is None:
-            mask = np.full((len(points['x']),), True)
+        if mask is None:  # TODO dit wordt 2 keer uitgevoerd (ook in process_file)
+            mask = np.full(len(points['x']), True)
+
+        if self.fusers:  # TODO kan dit mooier?
```

Comment:
Als we alleen region growing doen moet dit niet uitgevoerd worden...

Reply:
`if fusers is not None:`

en dan daar ook de for loop onder laten vallen?

Reply:
En hetzelfde voor de growers.